katello-installer/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-#global prever .rc1
+# %%global prever .rc1
```

Comment:
@ehelms note, i updated this and supposedly this will work, curious your thoughts.  I think i like this better than just '#global' myself